libraries/chain/kv_context_rocksdb.cpp

```diff
@@ -275,8 +301,8 @@ namespace eosio { namespace chain {
          const char* temp      = nullptr;
          uint32_t    temp_size = 0;
          if (temp_data_buffer) {
-            temp      = temp_data_buffer->data();
-            temp_size = temp_data_buffer->size();
+            temp      = temp_data_buffer->data() + kv_payer_size;
+            temp_size = temp_data_buffer->size() - kv_payer_size;
```

Comment:
set temp_size using static method to calculate actual size (and will prevent temp_size from being set with a negative value